### PR TITLE
Run ALTER TYPE migrations without a transaction

### PR DESCRIPTION
**Pull request by courte**
*\~quietly sobbing into elbow~*

I had to break out of SQLAlchemy to accurately update an enum. I also scraped out / reused some old code from those good ol' 2.0 migration days.

### CODE REVIEW

File: scripts.py [hunk 1]
```diff
@@ -1901,19 +1901,52 @@ def _run_migration(self, migration_path, timestamp):
...
+        # database: no transactions, no guardrails.
+        for command in sql_commands:
+            connection.execution_options(isolation_level='AUTOCOMMIT')\
+                .execute(text(command))
+
+        # Update the script's Session to a new one that has the changed schema
+        # and other important info.
+        self._session = Session(connection)
+        self.load_configuration()
+        DataSource.well_known_sources(self._db)
```
Comment (leonardr) [inline scripts.py]: Is this enough or should you go all out and call initialize_data?

Comment (courte) [reply scripts.py]: I'm not sure on a deep leve, but in testing `initialize_data` raised an error because the MaterializedViews already exist.

Comment (leonardr) [reply scripts.py]: OK, that makes sense. Let's see what happens.

Comment (courte) [reply scripts.py]: Oh, that was actually the initialization of the engine and the collection. I'll try this.

**Pull request merged**